Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -440,22 +523,31 @@ def chrome_manager() -> tuple[Any | None, str | None]:
         tuple[Any | None, int | None]: A tuple containing:
             - The Browser or None if an error occurred.
             - The chrome port or None if an error occurred.
+    chrome_port: {
+                        'instance_id': instance_id,
+                        'chrome_options': chrome_options,
+                        'chrome_rasterize_connections': '1'
+                    }
+    '''
     """
     # If instance_id or chrome_options are not set, assign 'None' to these variables.
     # This way, when fetching the content from the file, if there was no instance_id or chrome_options before,
     # it can compare between the fetched 'None' string and the 'None' that assigned.
     instance_id = demisto.callingContext.get('context', {}).get('IntegrationInstanceID', 'None') or 'None'
     chrome_options = demisto.params().get('chrome_options') or 'None'
-    chrome_instances_contents = read_file(CHROME_INSTANCES_FILE_PATH)
-    instance_id_to_chrome_options, instance_id_to_port, instances_id, chromes_options = \
-        get_chrome_instances_contents_dictionaries(chrome_instances_contents)
-
+    demisto.debug(f'MAI MORAG TEST {chrome_options}')
+    chrome_instances_contents = read_text_file(CHROME_INSTANCES_FILE_PATH) or {}
+    instances_id = []
+    for chrome_port_data in chrome_instances_contents.values():
+        instances_id.append(chrome_port_data['instance_id'])
     if not chrome_instances_contents or instance_id not in instances_id:
         return generate_new_chrome_instance(instance_id, chrome_options)
 
     elif chrome_options != instance_id_to_chrome_options.get(instance_id):
         chrome_port = instance_id_to_port.get(instance_id, '')
-        delete_row_with_old_chrome_configurations_from_chrome_instances_file(chrome_instances_contents, instance_id, chrome_port)
+        edit_chrome_port_file_configurations(chrome_port=chrome_port,
```

Comment:
in this case, the chrome_options is what you need to change.
lets say the file looks like that:
old_port = {instance_id, old_chrome_option, rast_count}
now user changes chrome port and we enter this case, we need to delete the old object from the file, create a new port and write to the file:
new_port = {same_instance_id, new_chrome_option, 1}

so here we will not need to update the exist port. we need to delete this port and create new one